docs/db_overview.md

```diff
@@ -53,4 +59,32 @@ The database contains four tables: one primary and three secondary tables. Each
 |config| An AEPsych Config object, stored as a Python pickle for easy serialization/deserialization.  (This may change in the future as Python pickle objects may not load across versions of Python or AEPsych). |
 |master_table_id| This is a reference key to the original entry in the master table's unique ID.
 
+- **Raw Table**: Fact table to store raw data for each iteration of a given experiment. It has two associated dimension tables, one for the parameters and one for the outcomes. The columns are described as followed:
+
+|Raw Table Column | Purpose |
+|:-: | :- |
+| unique_id | A unique ID generated for each entry in this table. |
+| timestamp | A timestamp indicating when the server received a tell message, and recorded the data of one iteration of a given experiment.|
+| master_table_id | This is a reference key to the original entry in the master table's unique ID. |
+| parameter_id | This is a reference key to the original entry in the parameter table's unique ID. |
+| model_data | Wheter or not the data from that iteration was modeled during the experiment.|
```

Comment:
Can you add examples of two SQL queries here: 
1. For a single experiment ID, construct a table whose headers are parameters, outcomes, and metadata items, and each row has one trial. 
2. Given a set of outcomes and/or parameters, construct a table whose headers are parameters, outcomes, and experiment ID, and each row is one trial. 

There should probably be convenience functions or methods in AEPsych to do both of these tasks which are the most common tasks we expect users to want ("get me all the data from experiment X" and "get me all the data from experiments which contain fields Y"). But adding these functions can be a separate PR.

Reply:
Hi!
I just realized that there is already a function in AEPsych for point 1! It's the `generate_experiment_table` method in the `server.py` file!
However, I'll create examples for both points with SQL and raw Pandas (without needing to use AEPsych). If there is a way to do it without Pandas (and Python), then I'll update the examples 😄

Reply:
The first version of the examples is ready! Let me know what you think about them :)